lib/builders-with-pkgs.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, pkgs }:
+with lib.nixvim.builders;
```

Comment:
Think we could be remove the `with ...` and be specific with inherits or inline?

Reply:
In this case we know what we're importing with the `with` statement, and we happen to be using every single attr from `lib.nixvim.builders`...

I'm fine keeping it, but I appreciate `with` can give off some funky smells 😅

Reply:
Ah, yeah, you're right. I didn't get to the builder.nix until after this part and didn't realize it was literally overriding everything from there. So, it is probably safe in this usage. I've just noticed it basically breaks LSPs ability to detect missing scopes when a `with` is present so I hate seeing them, now. :P

Reply:
I replaced the `with` with an `inherit (lib.nixvim) builders` to be more explicit. But you'll never see it because I then removed the file entirely in some refactoring! :sweat_smile: